azure-pipelines.yml

```diff
@@ -52,6 +52,7 @@ resources:
 
 # The pipeline is run under two primary conditions: if cutting a release or as scheduled by the above cron job.
 variables:
+  isReleasePrep: $[contains(variables['Build.SourceBranch'], 'release')]
```

Comment:
As it stands, our release process works as follows:
* Create a prep PR with changelogs and other misc updates
  - This triggers `dgtest`, which doesn't pick up any tests, and always the prep PR to merge 100% of the time
* Tag that particular prep PR commit to cut the release
  - This triggers `pytest`, which will run the full suite and validate that our build is not broken.

The issue here is what if the build is actually broken when we go to tag? We've already updated our changelogs and our git history now has this release commit that isn't actually the release. The proposed change here will run the full `pytest` suite on both the release PR and the tag (duplicative but safer).